admin/admin.go

```diff
@@ -80,9 +80,10 @@ func New(ctx context.Context, opts *Options, logger *zap.Logger, issuer *auth.Is
 	if opts.MetricsProjectOrg != "" && opts.MetricsProjectName != "" {
 		proj, err := db.FindProjectByName(ctx, opts.MetricsProjectOrg, opts.MetricsProjectName)
 		if err != nil {
-			return nil, fmt.Errorf("error looking up metrics project: %w", err)
+			logger.Sugar().Warnf("error looking up metrics project: %w", err)
```

Comment:
This feels slightly like an anti-pattern. It's cleaner to fail if an invalid value is provided (and then handle the project-not-created-yet case by not setting the env var until later).

But if this is really needed – can we at least change this log to error-level? (Since it's a real error that might indicate a misconfigured system.)